README.md

```diff
@@ -165,78 +184,566 @@ and then destroy them, all from the command line:
 puppet resource ec2_securitygroup test-group ensure=absent region=sa-east-1
 ```
 
+### Managing AWS Infrastructure
+
+You can use the aws module to audit AWS resources, launch autoscaling groups in VPC, perform unit testing, and more. The [examples](examples/) directory contains a variety of usage examples that should give you an idea of what's possible:
+
+* [Puppet Enterprise](puppet-enterprise/): Start up a small Puppet Enterprise cluster using the AWS module.
```

Comment:
the links here won't work as they're relative. You'll need to add the examples prefix to the URL